debian/nfd-init.service

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2017,  Endless Mobile, Inc.
+#
+# This file is NOT part of NFD (Named Data Networking Forwarding Daemon).
+#
+# NFD is free software: you can redistribute it and/or modify it under the terms
+# of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# NFD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# NFD, e.g., in COPYING.md file.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Author: Niv Sardi <xaiki@endlessm.com>
+
+[Unit]
+Description=NDN Forwarding Daemon post-init script
+Documentation=man:nfd man:nfdc man:nfd-status
+Wants=nfd.service
+After=nfd.service
+
+[Service]
+ConditionPathExists=/etc/ndn/nfd-init.sh
+Environment=HOME=/var/lib/ndn/nfd
+ExecStart=/bin/sh -ec 'sleep 2; . /etc/ndn/nfd-init.sh;'
+Type=oneshot
+RemainAfterExit=yes
+
+[Install]
+WantedBy=nfd.service
```

Comment:
I don't think this is right. nfd itself does not want or need this unit. If you want to link them together, I think it would be better to have `Also=nfd-init.service` in nfd.service so that enabling nfd.service also enables this unit. Then have this one `WantedBy=multi-user.target`. Personally, I wouldn't want this service enabled ever, so I'd just change it to `WantedBy=multi-user.target` and let the admin decide that they would want to turn this on.

You're also missing a newline at the end of the file.